quinn-proto/src/token.rs

```diff
@@ -317,6 +408,31 @@ mod test {
         assert_eq!(issued_1, issued_2);
     }
 
+    #[test]
+    fn validation_token_sanity() {
+        use crate::{Duration, UNIX_EPOCH};
+
+        use std::net::Ipv6Addr;
+
+        let ip_1 = Ipv6Addr::LOCALHOST.into();
+        let issued_1 = UNIX_EPOCH + Duration::new(42, 0); // Fractional seconds would be lost
```

Comment:
Prefer `Duration::from_secs` for readability.

Reply:
#2120